testdata/react_with_loop/dist/src/Running.js

```diff
@@ -0,0 +1,10 @@
+
+// a script to keep running forever
+
+let num = 0;
+export function runForever() {
+    setTimeout(() => {
+        num++;
+        runForever();
```

Comment:
Could use `setInterval` too

Reply:
I copy these files from the existing react project. I'm inclined to leave it as-is unless we have a good reason to change them